src/components/subscriptions/data/service.js

```diff
@@ -113,6 +131,21 @@ export function sendLicenseReminder(options = {}) {
  * call to get this data.
  */
 export function sendLicenseRevoke(options = {}) {
-  return Promise.resolve(options);
+  const { user } = options;
+  updateUserLicenseStatus({ user, license: DEACTIVATED });
+  const updatedUser = users.find(item => item.userId === user.userId);
+  return Promise.resolve(updatedUser);
+  // return Promise.reject(new Error('Could not connect to the server'));
+}
+
+/**
+ * This function mocks out the response from a non-existant API endpoint. Once the endpoint
+ * exists, the contents of this function will use the `apiClient` to make an actual API
+ * call to get this data.
+ */
+export function addUsers(options = {}) {
```

Comment:
adding this function in case it's useful for prototyping the "Add users" modal submission logic